docker-compose.yml

```diff
@@ -1,50 +1,42 @@
-version: '3'
+version: '3.9'
 
 services:
     
   db:
     image: postgres
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
+      - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
     restart: always
-    env_file:
-      - .env
     environment:
-      POSTGRES_USER: ${DB_USERNAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
     ports:
       - 5432:5432
-    expose:
-      - "5432"
 
   api:
     build:
       context: .
-      dockerfile: Docker_rails
+      dockerfile: Dockerfile
     volumes:
       - .:/app
-    command: /bin/sh -c "cd app && rm -f /app/tmp/pids/server.pid && bundle exec rails s -p 3001 -b 0.0.0.0"
     stdin_open: true
     tty: true
     restart: always
     env_file:
       - .env
     ports:
       - 3001:3001
-    links: 
-      - db
     depends_on: 
       - db
 
   front:
     build:
-      context: .
-      dockerfile: Docker_react
-    command: /bin/sh -c "cd client && yarn && yarn start"
+      context: client
+      dockerfile: Dockerfile
```

Comment:
Since the `front` container only needs the `client` directory, we can specify the context like this. 
And by putting the react Dockerfile inside the `client` directory, we can name both dockerfiles `Dockerfile` (yay)